resources/user-agents/browsers/mobile-safari/mobile-safari-8-x.json

```diff
@@ -35,27 +35,35 @@
             "iPod": "iPod Touch"
           },
           "platforms": [
-            "iOS_A_6_0", "iOS_A_6_1", "iOS_A_7_0", "iOS_A_7_1", "iOS_A_8_0", "iOS_A_8_0_B",
```

Comment:
There's no change of versions here, I just was re-organizing a bit to get like-Major versions on the same line.